tests/queries/0_stateless/02784_connection_string.sh

```diff
@@ -0,0 +1,159 @@
+#!/usr/bin/env bash
```

Comment:
Do the tests run on different machines? 

If not, we should not have parallel problems.

This code create unique_users. `$$` means Process ID. It should be unique for every running shell and other process.
``` bash
TEST_USER_NAME="test_user_02771_$$"
TEST_USER_EMAIL_NAME="test_user_02771_$$@some_mail.com"
TEST_USER_EMAIL_NAME_ENCODED="test_user_02771_$$%40some_mail.com"
```

In case if tests run on different machines simultaneosuly, then it would be possible to get same PID on different machines. And I have to add `Tags: no-parallel` (and remove `$$` from the test code)

Reply:
I missed that the process ID is part of the user name. We should be fine then.